ietf/meeting/management/commands/populate_attended.py

```diff
@@ -0,0 +1,76 @@
+# Copyright The IETF Trust 2022, All Rights Reserved
+
+import debug # pyflakes: ignore
+
+from tqdm import tqdm
+
+from django.core.management.base import BaseCommand
+
+#from ietf.meeting.models import Meeting, Session
+from ietf.meeting.models import Session
+#from ietf.meeting.helpers import get_schedule
+from ietf.meeting.utils import sort_sessions
+from ietf.person.models import Person, Email
+
+import json
+
+class Command(BaseCommand):
+
+    help = 'Populates the meeting Attended table based on bluesheets and registration information'
+
+    def add_arguments(self, parser):
+        parser.add_argument('filename', nargs='+', type=str)
+
+    def handle(self, *args, **options):
+
+        issues = []
+        session_cache = dict()
+        skipped = 0
+        for filename in options['filename']:
+            records = json.loads(open(filename,'r').read())
+            for record in tqdm(records):
+                user = record['sub']
+                session_acronym = record['group']
+                meeting_number = record['meeting']
+                email = record['email']
+                order = 0
```

Comment:
Maybe worth a comment about what's up with the `order` parameter here? It's puzzling to me, anyway.

Reply:
done